File: IsometricShooter/Scripts/JS/GUI/HealthGui.js
#pragma strict

var playerHealth : HealthConf; //Health reference of player
var health : UI.Text; //GUI text to show health

function OnGUI () {

	//health.text = playerHealth.currentHealth + " / " + playerHealth.maximumHealth;
	health.text = "" + playerHealth.currentHealth;

}

File: IsometricShooter/Scripts/CS/GUI/HealthGui.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class HealthGui : MonoBehaviour {

	public HealthConf playerHealth; //Health reference of player
	public Text health; //GUI text to show health

	void OnGUI () {

		//health.text = playerHealth.currentHealth + " / " + playerHealth.maximumHealth;
		health.text = "" + playerHealth.currentHealth;

	}
	
}
